### ISSUE TITLE
creating a react website for this repo.

### ISSUE THREAD

**Issue by amansrivastava905** [swapagarwal/awesome-eli5#53]
I can create a simple responsive website using react, styled-components & a bit of material-UI.
It may have features like sorting based upon topic,
to contribute a new item, you simply have to update the data.js file which will store content in form of objects.
If you have thought of any other flow, please comment. Will try to incorporate them.

I will share updates here.

_Originally posted by @amansrivastava905 in https://github.com/swapagarwal/awesome-eli5/issues/19#issuecomment-805854631_

**Comment by swapagarwal** [comment]
Sounds good!
